lab-guide/2-implement-simple-color-based-segmentation.md

```diff
@@ -23,10 +23,11 @@ The method `MultivariateNormalModel::performTraining` should estimate the mean `
 It must also compute the inverse of the covariance matrix `inverse_covariance`, which we later will use to compute the Mahalanobis distance.
 
 Take a look at [cv::calcCovarMatrix](https://docs.opencv.org/4.0.1/d2/de8/group__core__array.html#ga017122d912af19d7d0d2cccc2d63819f).
-We want `mean_` to be a column vector. The [cv::CovarFlags](https://docs.opencv.org/4.0.1/d0/de1/group__core.html#ga719ebd4a73f30f4fab258ab7616d0f0f) in `cv::calcCovarMatrix()` can be combined like this: `flag1|flag2`.
+We want `mean_` to be a row vector (to avoid having to transpose the matrix of samples). 
```

Comment:
En del av endringene i dokumentasjonen, slik som denne, må også reverseres.

Reply:
Da har jeg fikset denne og gjort noen andre små justeringer på teksten.

Gitt at jeg valgte å ikke endre koden i lab_11_solution, så tror jeg denne er klar for merge.